### PR TITLE
add switch for hide system pods

### PR DESCRIPTION
**Pull request by giovannyreyeso**
[ADD FEATURE] - Support hide/show system pods

### CODE REVIEW

File: dashboard/components/Node.tsx [hunk 1]
```diff
@@ -130,7 +131,12 @@ const renderResources = (container: Container) => {
 
 const PodComponent = ({ entity, opts }: NodeProps) => {
   console.log(entity)
+  const hideSystemPod = opts.hideSystemPod;
   const bg = getColor(entity)
+  debugger
```
Comment (stephenlacy) [inline dashboard/components/Node.tsx]: Could you remove the debugger call?

**Review by stephenlacy — CHANGES_REQUESTED** [review]

**Comment by stephenlacy** [comment]
Thanks for the PR! Would you be able to remove the debugger call before I merge? thanks

File: dashboard/components/Node.tsx (hunk 1 above)
Comment (giovannyreyeso) [reply dashboard/components/Node.tsx]: Sure!.  is done!

**Comment by giovannyreyeso** [comment]
> Thanks for the PR! Would you be able to remove the debugger call before I merge? thanks

i removed this line in other commit

**Pull request merged**